src/OrchardCore.Modules/OrchardCore.Title/Views/TitlePart.Edit.cshtml

```diff
@@ -1,9 +1,17 @@
-@using OrchardCore.Title.ViewModels;
 @model TitlePartViewModel
+@using OrchardCore.ContentManagement
+@using OrchardCore.ContentManagement.Metadata.Models
+@using OrchardCore.Localization
+@using OrchardCore.Title.ViewModels;
+@inject IContentManager ContentManager
+
+@{
+    var cultureAspect = await ContentManager.PopulateAspectAsync<CultureAspect>(Model.TitlePart.ContentItem);
```

Comment:
We could also add an extension method to the `Orchard` razor helper that would return the culture info of a content item. This way you don't have to inject the CM everywhere.

Reply:
This is what I wanna ask you before completing the PR